include/openenclave/enclave.h

```diff
@@ -636,6 +636,40 @@ oe_result_t oe_generate_attestation_certificate(
     uint8_t** output_cert,
     size_t* output_cert_size);
 
+/**
+ * oe_generate_plugin_attestation_certificate.
+ *
+ * This function generates a self-signed x.509 certificate with an embedded
+ * report generated by a plugin attester from the underlying enclave.
+ *
+ * @param[in] format_id format id for the plugin attester
+ *
+ * @param[in] subject_name a string contains an X.509 distinguished
+ * name (DN) for customizing the generated certificate. This name is also used
+ * as the issuer name because this is a self-signed certificate
+ * See RFC5280 (https://tools.ietf.org/html/rfc5280) specification for details
+ * Example value "CN=Open Enclave SDK,O=OESDK TLS,C=US"
+ *
+ * @param[in] private_key a private key used to sign this certificate
+ * @param[in] private_key_size The size of the private_key buffer
+ * @param[in] public_key a public key used as the certificate's subject key
+ * @param[in] public_key_size The size of the public_key buffer.
+ *
+ * @param[out] output_cert a pointer to buffer pointer
+ * @param[out] output_cert_size size of the buffer above
+ *
+ * @return OE_OK on success
+ */
+oe_result_t oe_generate_plugin_attestation_certificate(
```

Comment:
+1 to preferring "get" instead of "generate"

Reply:
One additional observation, this PR will conflict with #3198, which changes `evidence_format` to `format_id` and also brings in the API change to `oe_verify_evidence` to add the explicit `format_id` parameter.

Reply:
@CodeMonkeyLeet thanks for catching the conflict. A few files in the `include` and `tests` directories are also changed in PR #3198, and likely will have conflicts.

@jxyang since this PR will invoke the new APIs implemented in PR #3198, can you wait till that PR is merged, and rebase to it?

Reply:
@shnwc Sure. Looks like your PR has more than enough approvals. What's stopping it from being merged?

Reply:
PR #3198 is merged now. Changes in public API takes longer go reach agreement.

Reply:
Thanks. I will rebase.

Reply:
Why does #3198 changes `evidence_format` to `format_id`? Should this API keep using `format_id`?